src/ViewModel/SocialMediaSharers.php

```diff
@@ -12,7 +12,7 @@
 
 final class SocialMediaSharers implements ViewModel
 {
-    const TWITTER_LENGTH = 140 - 1;
+    const TWITTER_TITLE_LENGTH = 140 - 1 - 23; // Max - space - URL length
```

Comment:
not sure if https://techcrunch.com/2017/11/07/twitter-officially-expands-its-character-count-to-280-starting-today/ should be official and universal by now?

Reply:
Features team confirms that all that this should be true for everyone, it would likely make titles always fit.
Longest title we have is `Correction: The transforming growth factor beta signaling pathway is critical for the formation of CD4 T follicular helper cells and isotype-switched antibody responses in the lung mucosa` which is 187 characters.

Reply:
Twitter docs seem to avoid the length for some reason (https://developer.twitter.com/en/docs/basics/counting-characters), but it's defined in https://github.com/twitter/twitter-text/blob/4b8fc4a69d5a24ec002c10acda3e77967e408a57/config/v2.json#L3

Reply:
Might as well remove the limit, since we're a way off it. It's unlikely we're going to hit the limit in the future too, but the end user would just have to truncate it themselves to post.